_ja/overviews/collections/conversions-between-java-and-scala-collections.md

```diff
@@ -25,8 +25,8 @@ Scala と同様に、Java
 
 このような変換を作動させるには、[JavaConverters](https://www.scala-lang.org/api/{{ site.scala-version }}/scala/collection/JavaConverters$.html) オブジェクトからインポートするだけでいい:
 
-    scala> import collection.JavaConverters._
-    import collection.JavaConverters._
+    scala> import scala.jdk.CollectionConverters._
+    import scala.jdk.CollectionConverters._
```

Comment:
Only the overview for the scala 2.13 collections should be changed I think, the overview for the old collections should stay as is.